cdk/lib/security-hq.ts

```diff
@@ -0,0 +1,17 @@
+import { CfnInclude } from '@aws-cdk/cloudformation-include';
```

Comment:
This is the file with the actual CDK code for Security HQ. At the moment, it simply includes the existing template.